src/main/java/core/basesyntax/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Inventory;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.strategy.FruitOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
The class `ShopServiceImpl` should be placed in the `service` package, not in the `service.impl` package according to the checklist requirement to keep logic separated in corresponding packages.